spec/views/hyrax/admin/collection_types/index.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+RSpec.describe 'hyrax/admin/collection_types/index.html.erb', type: :view do
+  before do
+    assign(:collection_types, [
+             FactoryGirl.create(:collection_type, title: 'Test Title 1'),
+             FactoryGirl.create(:collection_type, title: 'Test Title 2')
```

Comment:
@elrayle @kdid Thanks for the feedback.  I'll make these adjustments now and see if we can get it green.